### PR TITLE
Release 5.5.0

### PR DESCRIPTION
**Pull request by fredericDelaporte**

### CODE REVIEW

**Review by hazzik — DISMISSED** [review]

**Comment by fredericDelaporte** [comment]
MySql AppVeoyor build is failing as if the database was no more available on the build image. See appveyor/ci#3895.

**Comment by bahusoid** [comment]
#3438 looks like 5.3 regression which would be nice to fix before release

**Comment by fredericDelaporte** [comment]
So, is this waiting for proper support of DB2/400 schema, or something else?

**Comment by hazzik** [comment]
I don’t think so. 
(I’ve merged NUnit 3.14 though, so the list needs to be updated)

**Comment by fredericDelaporte** [comment]
I did forget about it. Now that is rebased and added in the release notes.

**Review by hazzik — DISMISSED** [review]

**Comment by wilbit** [comment]
Looking forward to see the release 😊

File: releasenotes.txt [hunk 1]
```diff
@@ -1,4 +1,94 @@
...
+** Bug
+
+   * #3413 Downgrade dependency System.Data.SQLite.Core 1.0.118 -> 1.0.117
+   * #3406 Fix orphan removal for detached one-to-one
+   * #3392 Partial fix fetching lazy property after Select in Linq
+   * #3360 Incorrect parameter length for char types in MicrosoftDataSqlClientDriver
+   * #3334 Exception executing HQL query with uncorrelated left joins in subselect
+   * #3327 HqlParser does not correctly negate EXISTS-nodes below an AND/OR
+   * #3325 Cascading orphan delete may not work on versioned entity
+   * #3311 NamedQuery igonres <query-param type="AnsiString"> and any other
```
Comment (wilbit) [inline releasenotes.txt:22]:
```suggestion
   * #3311 NamedQuery ignores <query-param type="AnsiString"> and any other
```

**Review by hazzik — APPROVED** [review]

**Pull request merged**